apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -18,7 +18,7 @@ const GLOBAL_QUERY_LIMIT = 1000
 
 class GqlWithKnexLoadList {
 
-    constructor ({ listKey, fields, singleRelations = [], multipleRelations = [], where = {}, sortBy = [] }) {
+    constructor ({ listKey, fields, singleRelations = [], multipleRelations = [], where = {}, sortBy = [], offset, limit }) {
```

Comment:
you don't use this params inside constructor. Are you sure that you really need it?